src/hott/06-contractible.rzk.md

```diff
@@ -206,28 +206,30 @@ A retract of contractible types is contractible.
 #section retraction-data
 
 #variables A B : U
-#variable AretractB : is-retract-of A B
+#variable is-retract-of-A-B : is-retract-of A B
 
 #def is-retract-of-section
   : A → B
-  := first AretractB
+  := first is-retract-of-A-B
 
 #def is-retract-of-retraction
   : B → A
-  := first (second AretractB)
+  := first (second is-retract-of-A-B)
 
 #def is-retract-of-homotopy
   : homotopy A A (composition A B A is-retract-of-retraction is-retract-of-section) (identity A)
-  := second (second AretractB)
+  := second (second is-retract-of-A-B)
+```
 
--- If A is a retract of a contractible type it has a term.
-#def is-retract-of-is-contr-isInhabited uses (AretractB)
+```rzk title="If $A$ is a retract of a contractible type it has a term"
+#def is-retract-of-is-contr-isInhabited uses (is-retract-of-A-B)
   ( is-contr-B : is-contr B)
   : A
   := is-retract-of-retraction (contraction-center B is-contr-B)
+```
 
--- If A is a retract of a contractible type it has a contracting homotopy.
-#def is-retract-of-is-contr-hasHtpy uses (AretractB)
+```rzk title="If $A$ is a retract of a contractible type it has a contracting homotopy"
```

Comment:
Same as above:

```suggestion
```rzk title="If A is a retract of a contractible type it has a contracting homotopy"
```